metrics-operator/main.go

```diff
@@ -137,6 +141,23 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "KeptnMetric")
 		os.Exit(1)
 	}
+
+	analysisLogger := ctrl.Log.WithName("KeptnMetric Controller")
+	targetEval := analysis.NewTargetEvaluator(&analysis.OperatorEvaluator{})
+	objEval := analysis.NewObjectiveEvaluator(&targetEval)
+	analysisEval := analysis.NewAnalysisEvaluator(&objEval)
+
+	if err = (&analysiscontroller.AnalysisReconciler{
```

Comment:
we should hide the controller under feature flag, as it's not ready for usage yet

Reply:
Yes will add soon an env var in the helm chart/main